src/components/search-modal/search-badge.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {
+  Badge,
+  IconButton
+} from '@folio/stripes-components';
+
+import styles from './search-badge.css';
+
+export default function SearchBadge({ filterCount = 0, onClick, ...rest }) {
+  return (
+    <div className={styles['search-filter-area']} {...rest}>
+      {filterCount > 0 && (
+        <span data-test-eholdings-search-filters="badge">
+          <Badge className={styles['filter-count']}>{filterCount}</Badge>
+        </span>
+      )}
+      <IconButton
+        icon="search"
+        onClick={onClick}
+        data-test-eholdings-search-filters="icon"
+      />
```

Comment:
In this situation...
- Apply a filter
- Click the search icon to collapse the leftmost pane
- There's now a number of filters count next to the icon
.. .the search icon has a CSS problem where the inner part of the header overlaps it, making very little of the icon clickable (discoverable by hovering over the icon).

Looks like we're supposed to use the `badgeCount` prop on `<IconButton>` instead: https://github.com/folio-org/stripes-smart-components/blob/0565ade001de925a0df27a0be31d1a71b3070451/lib/SearchAndSort/SearchAndSort.js#L512-L517